src/DIRAC/Resources/Computing/SingularityComputingElement.py

```diff
@@ -48,18 +46,23 @@
 
 CONTAINER_WRAPPER_INSTALL = """#!/bin/bash
 
-echo "Starting inner container wrapper scripts (uses dirac-install) at `date`."
+echo "Starting inner container wrapper scripts at `date`."
```

Comment:
Perhaps we also want to set -e here so that the script stops if it fails the curl or any other step? Not sure if that'll break it in some other way...